azure-configurationservice/azure/configuration/__init__.py

```diff
@@ -0,0 +1,23 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from .azure_configuration_client import AzureConfigurationClient
+from .azure_configuration_client_abstract import AzureConfigurationClientAbstract
+from ._generated.models import ConfigurationSetting
+
+__version__ = "0.1"
+__all__ = [
+    'AzureConfigurationClient',
+    'ConfigurationSetting',
+    'AzureConfigurationClientAbstract'
```

Comment:
Why does AzureConfigurationClientAbstract have to be included in __all__? Is this only for the doc process to pick up dockstrings?

Reply:
Should be a mistake. I'll remove it